test/functional/repository/create_test.rb

```diff
@@ -0,0 +1,41 @@
+require File.join(File.dirname(__FILE__), '../test_helper')
+
+describe "create repository" do
+  let(:org_id) { 1 }
+  let(:product_id) { 2 }
+  let(:name) { "repo1" }
+  let(:content_type) { "yum" }
+
+  it 'with basic options' do
+    api_expects(:repository, :create, 'create a repository') do
+      with_params(
+        'product_id' => product_id,
+        'content_type' => content_type,
+        'name' => name)
+    end
+
+    command = %W(repository create --organization-id #{org_id} --product-id #{product_id}
+                 --content-type #{content_type} --name #{name})
+
+    assert_equal(0, run_cmd(command).exit_code)
+  end
+
+  it 'with ssl options by name' do
+    api_expects(:repository, :create, 'create a repository') do
+      with_params(
+        'organization_id' => org_id,
+        'product_id' => product_id,
+        'content_type' => content_type,
+        'ssl_ca_cert_id' => 1,
+        'ssl_client_cert_id' => 2,
+        'ssl_client_key_id' => 3,
+        'name' => name)
+    end
+
+    command = %W(repository create --organization-id #{org_id} --product-id #{product_id}
+                 --content-type #{content_type} --name #{name} --ssl-client-cert-name test_cert
+                 --ssl-client-key-name test_key --ssl-ca-cert-name test_ca)
```

Comment:
This doesn't quite match other options from other commands. The `-name` portion of an option is usually implied if we leave off the field, so it maybe should accept `--ssl-client-cert`, `--ssl-client-key`, `--ssl-ca-cert`.